config/base/datahub.yml.tmpl

```diff
@@ -6,21 +6,9 @@ source:
     manifest_path: "./target/manifest.json"
     catalog_path: "./target/catalog.json"
     sources_path: "./target/sources.json"
-
     target_platform: "bigquery"
-    disable_dbt_node_creation: True
-    load_schemas: True
```

Comment:
Why these two are deleted?

Reply:
These flags are deleted because of lacking lineage between Postgres and dbt ingestion 

Rather than having this:
![image](https://user-images.githubusercontent.com/58829617/177712409-eed5cacc-7cf0-413b-b94a-cd7eedad4a4d.png)
![image](https://user-images.githubusercontent.com/58829617/177712467-482eccb0-a357-42db-9265-5c66edbdff9b.png)
We are getting this:
![image](https://user-images.githubusercontent.com/58829617/177712524-1a83cba5-0dd0-4988-a103-581ee0fc34c1.png)

Duplication of tables

with flag `disable_dbt_node_creation`
![image](https://user-images.githubusercontent.com/58829617/177716284-8953beac-1d6a-49ee-a4e2-e4a12ed3a53c.png)

without flag `disable_dbt_node_creation`
![image](https://user-images.githubusercontent.com/58829617/177715387-c5f76ff1-ff3b-42cf-a8bd-10b75b82ba80.png)